cfme/tests/services/test_service_catalogs.py

```diff
@@ -26,6 +26,7 @@
 ]
 
 
+@pytest.mark.rhv3
```

Comment:
This feel strange when compared to the tier of test_tagdialog_catalog_item. This test case is more basic and essential and has a lower tier

Reply:
Well this test functionality is tested in the test test_order_catalog_item_via_rest (that follow it, see right bellow).
It is not set to T3 compared to test_tagdialog_catalog_item. It is set to T3, as there is already another test that was picked, and check the functionality.

Reply:
Once again, I suggest switching tiers - UI before REST.

Reply:
OK, I will switch the tiers between these 2 tests.